src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -108,27 +108,39 @@
             </trans-unit>
             <trans-unit id="27">
                 <source>This value is not a valid URL.</source>
-                <target>Värdet är inte en giltig URL.</target>
+                <target>Värdet är inte en giltig webbadress.</target>
```

Comment:
shouldn't we keep the previous translation? URL is more accurate than "web address"

Reply:
Yeah. I know these could be a bit technical. 

I tried to write "domän" for the english "host" once. But was corrected. 

Let's keep URL.